onebusaway-webapp/src/main/java/org/onebusaway/webapp/impl/GWTMessagesFactory.java

```diff
@@ -33,36 +35,34 @@ public GWTMessagesFactory() {
 
   protected Object createInstance() throws IOException {
     String name = _type.getName();
-    name = "/" + name.replace('.', '/') + ".properties";
-    InputStream is = _type.getResourceAsStream(name);
-    if (is == null)
-      throw new IllegalStateException("Unable to find resources: " + name);
-    Properties p = new Properties();
-    p.load(is);
-    Handler handler = new Handler(p);
+    name = "/" + name.replace('.', '/');  
+    ReloadableResourceBundleMessageSource messageSource = new ReloadableResourceBundleMessageSource();
+    messageSource.setBasename(name);
```

Comment:
Here we have the base name (eg: ArrivalAndDepartureMessages) and find the locale resources:
* ArrivalAndDepartureMessages_en.properties
* ArrivalAndDepartureMessages_es.properties
* ArrivalAndDepartureMessages_fr.properties
* ArrivalAndDepartureMessages_{lang}.properties

But on this method dont use the default ArrivalAndDepartureMessages.properties file.